### PR TITLE
[libcgal_julia] Method sig fixes in v0.6.3

### PR DESCRIPTION
**Pull request by rgcv**
Since I'm using (and abusing) macros, I forget which implicitly create
lambdas with parameters that are either `const T&` or just plain `T`
(const ref vs. values).

Turns out it leads to issues when using mapped types in method
signatures, since it couldn't convert a plain julia type to a
corresponding C type when hitting `ccall`. Changing the parameter type
`T` to a `const T&` does the work. Don't think it had to be `const`,
just not a value-type.

In short: `Point23/Origin` interop (`==` testing, `Vector23` creation
from subtraction)

### CODE REVIEW

**Comment by rgcv** [comment]
Got a hot patch for this! Shouldn't be merged just yet, retagging very soon! ^^'
